examples/frameworks/nat_autogen_demo/src/nat_autogen_demo/register.py

```
@@ -14,4 +14,4 @@
 # limitations under the License.
 
 from . import autogen_team  # noqa: F401 # pylint: disable=W0611     #imported for side effects (registration)
-from . import weather_update_tool  # noqa: F401 # pylint: disable=W0611     #imported for side effects (registration)
+from . import traffic_status_tool  # noqa: F401 # pylint: disable=W0611     #imported for side effects (registration)
```

Ruff (0.14.11)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)